picard/ui/searchdialog.py

```diff
@@ -463,13 +463,13 @@ def display_results(self):
             track = obj[0]
             table_item = QtWidgets.QTableWidgetItem
             self.table.insertRow(row)
-            self.table.setItem(row, self.colpos('name'), table_item(track.get("title", "")))
-            self.table.setItem(row, self.colpos('length'), table_item(track.get("~length", "")))
-            self.table.setItem(row, self.colpos('artist'), table_item(track.get("artist", "")))
-            self.table.setItem(row, self.colpos('release'), table_item(track.get("album", "")))
-            self.table.setItem(row, self.colpos('date'), table_item(track.get("date", "")))
-            self.table.setItem(row, self.colpos('country'), table_item(track.get("country", "")))
-            self.table.setItem(row, self.colpos('type'), table_item(track.get("releasetype", "")))
+            self.set_table_item(row, 'name',    track, "title")
```

Comment:
Ditto here about PEP8

Reply:
> Ditto here about PEP8

Ditto
https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds ;)